sim/hunter/talents.go

```diff
@@ -160,3 +162,17 @@ func (hunter *Hunter) mortalShots() float64 {
 func (hunter *Hunter) trapMastery() float64 {
 	return 5 * float64(hunter.Talents.TrapMastery) * core.SpellHitRatingPerHitChance
 }
+
+func (hunter *Hunter) applyEfficiency() {
+	hunter.OnSpellRegistered(func(spell *core.Spell) {
+		// doesn't work on kill shot for whatever reason
+		if spell.SpellCode == SpellCode_HunterKillShot { 
+			return 
+		}
+
+		// applies to Stings, Shots, Strikes and Volley
+		if spell.Flags.Matches(SpellFlagSting) || spell.Flags.Matches(SpellFlagShot) || spell.Flags.Matches(SpellFlagStrike) || spell.SpellCode == SpellCode_HunterVolley {
```

Comment:
I think you can also do `spell.Flag.matches(SpellFlagSting | SpellFlagShot | SpellFlagStrike)` to simplify this a bit. Sorry just had the idea